### DISCUSSION TITLE
Missing Publication when trying to enable Realtime

### DISCUSSION THREAD

**Discussion by benoitmartel1** [supabase/supabase#8838] [Questions] [answered]
I just found out this morning I have no more realtime updates on my tables.
When I try to re-enable Realtime, I get the message "Missing publication".

In the Replication tab, I don't have the supabase_realtime entry anymore...

Anybody has any hint on what's going or how to get realtime working again?

**Comment by w3b6x9** [comment, marked as answer]
@benoitmartel1 you can re-create the publication by going to your sql editor and running the following query:

```
create publication supabase_realtime;
```

we discovered a rare issue when toggling all tables then only a subset where the publication is not created. we will be fixing this edge case very shortly.
